InertiaCore/ResponseFactory.cs

```diff
@@ -77,7 +77,7 @@ public async Task<IHtmlContent> Html(dynamic model)
                 ReferenceHandler = ReferenceHandler.IgnoreCycles
             });
 
-        var encoded = HttpUtility.HtmlEncode(data);
+        var encoded =  WebUtility.HtmlEncode(data);
```

Comment:
```suggestion
        var encoded = WebUtility.HtmlEncode(data);
```

Just remove the double space